### PR TITLE
Throwable param support

### PR DESCRIPTION
**Pull request by vietj**

### CODE REVIEW

File: src/main/java/io/vertx/codegen/ClassModel.java [hunk 1]
```diff
@@ -227,7 +227,8 @@ protected void checkParamType(Element elem, TypeMirror type, TypeInfo typeInfo,
     // Basic types, int, long, String etc
     // JsonObject or JsonArray
     // Also can use Object as a param type (e.g. for EventBus)
```
Comment (cescoffier) [inline src/main/java/io/vertx/codegen/ClassModel.java]: comment not updated after adding throwable.

File: src/main/resources/lang-type-mapping.properties [hunk 2]
```diff
@@ -16,6 +16,7 @@ JS.JSON_OBJECT=object
 JS.JSON_ARRAY=Array
 JS.API=api
 JS.OBJECT=ALL
+JS.THROWABLE=object
```
Comment (cescoffier) [inline src/main/resources/lang-type-mapping.properties]: there is actually an object Error in javascript (but it's an object - so ignore this comment)

File: README.md [hunk 3]
```diff
@@ -110,6 +110,7 @@ The following set `P` of types are permitted as parameters to any API method:
     * the set `J`
     * the set `V`    
 * any Enum class
+* the exact `java.lang.Throwable` class
```
Comment (cescoffier) [inline README.md]: Worth mentioning that Exception is not supported. Maybe a sentence like this:

`java.lang.Throwable` - must be this class exactly, `Exception` or sub types are not supported.

File: src/main/resources/lang-type-mapping.properties (hunk 2 above)
Comment (vietj) [inline src/main/resources/lang-type-mapping.properties]: yes it's for JS error object indeed that has typeof : object

File: README.md (hunk 3 above)
Comment (vietj) [inline README.md]: the code generator will yell at compile time :-) , fair enough

**Pull request merged**